AsyncDisplayKit/ASCellNode.mm

```diff
@@ -120,10 +121,10 @@ - (void)setLayerBacked:(BOOL)layerBacked
   ASDisplayNodeAssert(!layerBacked, @"ASCellNode does not support layer-backing.");
 }
 
-- (void)setNeedsLayout
+- (void)__setNeedsLayout
```

Comment:
There is also calculated layout to change, although I'm not entirely sure what the situation is in this change. Is this related to what people have been discussing on the open-source slack channel?

Sent from my iPad

> On Apr 21, 2016, at 12:55 PM, Adlai Holler <notifications@github.com> wrote:
> 
> In AsyncDisplayKit/ASCellNode.mm:
> 
> > @@ -120,10 +121,10 @@ - (void)setLayerBacked:(BOOL)layerBacked
> >    ASDisplayNodeAssert(!layerBacked, @"ASCellNode does not support layer-backing.");
> >  }
> >  
> > -- (void)setNeedsLayout
> > +- (void)__setNeedsLayout
> Hrm that may be a no-go since we don't get the old value in there.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>